levelkdev/swapr-dapp
# update coinbase branding

# Summary

Closes #1045

To test:
1. Open wallet selection modal
2. Verify that coinbase icon has inner white circle
3. Verify that wording has changed from 'Coinbase' to 'Coinbase Wallet'
